headers.js

```diff
@@ -72,9 +72,7 @@ var toTypeflag = function (flag) {
 }
 
 var alloc = function (size) {
```

Comment:
I think you can remove `alloc()` completely and replace the only use with `Buffer.alloc(512)`.